File: backend/Server/Server/Models/CartContent.cs
namespace Server.Models
{
    public class CartContent
    {

        public int Id { get; set; }
        public int ProductId { get; set; }
        public int Quantity { get; set; }
        public int ShoppingCartId { get; set; }

        public Product Product { get; set; }

    }
}

File: frontend/src/app/models/cart-content.ts
import { Product } from "./product";

export class CartContent {
    ProductId: number;
    Quantity: number;
    Product: Product
    constructor(productId: number, quantity: number, product: Product)
    {
        this.ProductId = productId;
        this.Quantity = quantity;
        this.Product = product;
    }
}
